BootCampV2/Homeworks/SecondWeek/ArrayExtension.cs

```diff
@@ -0,0 +1,149 @@
+using System;
+
+namespace BootCampV2
+{
+    public class ArrayExtension
```

Comment:
C# has a special feature called "Extension methods" that we haven't spoken about yet in class.  In general, files whose name ends in "Extensions" or "Extension" are adding extension methods.  Since you don't do that here you shouldn't name this file `ArrayExtension`.